tests/test_file_service.py

```
@@ -0,0 +1,392 @@
+            {'name': 'recent_file1.txt', 'path': '/path/to/recent_file1.txt', 'size': 100, 'mtime': recent_file_mtime},
+            {'name': 'recent_file2.txt', 'path': '/path/to/recent_file2.txt', 'size': 50, 'mtime': recent_file_mtime},
+        ]
+        assert cleanup_old_uploads(days=7) == 0
+        mock_delete_upload.assert_not_called()
+
+    # Async tests
+    @pytest.mark.asyncio
+    @patch('services.file_service.save_upload_file')
+    @patch('asyncio.get_running_loop')
+    @patch('services.file_service._get_thread_pool') # Added patch
+    async def test_save_upload_file_async(self, mock_get_running_loop, mock_save_upload_file, mock_upload_file, event_loop):
```

Ruff (0.14.0)

Unused method argument: `event_loop`

(ARG002)